default.nix

```diff
@@ -128,6 +129,33 @@ rec {
     import ./templates/project { inherit lib pkgs sources; }
   );
 
+  pre-commit-hook = pre-commit-hooks.run {
```

Comment:
Why is this repeating the invocation in flake.nix? It seems to use a slightly different interface there (selecting system via attribute) but still does pretty much the same.

Reply:
Because we can't define it here and use it in the flake as well (see comment)